lib/scout/view/mobile_scout.dart

```diff
@@ -103,7 +103,7 @@ class _MobileScoutState extends State<MobileScout> {
                             child: Row(
                               children: [
                                 Text(
-                                  'APT List',
+                                  'Live Markets',
```

Comment:
We already have a markets section. Change the name over there. From my understanding the scout page shows all the apts. The market page shows all the markets.

Reply:
can you point me to the live markets section, did you mean the prediction page?

Reply:
Ah yes that is what I meant. The predictions page that shows the listing of all the markets.